src/Symfony/Bridge/PhpUnit/Tests/ClockMockTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use Symfony\Bridge\PhpUnit\ClockMock;
+
+class ClockMockTest extends \PHPUnit_Framework_TestCase
+{
+    public function testTimeNowNull()
```

Comment:
As a general rule, I'd say code coverage should never be a target: behavior coverage should. Code coverage should be used as a way to detect where edge-case behavior tests are missing. Code coverage is only a consequence of good behavior coverage.